### DISCUSSION TITLE
Re-engagement error

### DISCUSSION THREAD

**Discussion by ghost** [chatwoot/chatwoot#9659] [Q&A]
I recently upgraded my seld hosted chatwoot from version 3.3.1 to 3.4.0. But now i get "Re-engagement error" on some of my conversations. I am still new to chatwoot and is unable to reproduce or understand what this error means. Have attached screenshot. Any heklp would be appreciated
<img width="448" alt="Screenshot 2024-06-20 at 1 10 12 PM" src="https://github.com/chatwoot/chatwoot/assets/137174882/ad9aee1b-b2cb-43dc-9bdf-2651b4f73044">

**Comment by aureliolk** [comment]
UP
